src/assets/css/fonts.css

```diff
@@ -1,102 +1,414 @@
 @font-face {
-  font-family: 'Arial';
-  src: url('../fonts/Arial/arial.ttf') format('truetype') font-weight-normal,
-       url('../fonts/Arial/arialbd.ttf') format('truetype') font-weight-bold,
-       url('../fonts/Arial/ariali.ttf') format('truetype') font-weight-italic,
-       url('../fonts/Arial/arialbi.ttf') format('truetype') font-italic font-weight-bold,
+  font-family: 'Courier New';
+  src: url('../fonts/../fonts/CourierNewPSMT.woff2') format('woff2'),
```

Comment:
What's with the replacement with `woff2` files?

Reply:
No idea, I used a website to generate the css. It works for me. Lmk if it doesnt work on someone's computer

Reply:
I would also rearrange the file to be alphabetical order for less confusion, and maybe add comments between each set of fonts

Reply:
The online auto geneartor I used didnt put the files in order or even together :( This was already me cleaning it up lol

Comments are definitely unneccessary, there is very little you can say here without being redundant.

Reply:
Right, but if you add a comment just stating the font name before each one it would help find them easier if scrolling through the file

Reply:
Dude the font name is literally in the font family